packages/tpetra/core/src/Tpetra_Details_makeColMap_def.hpp

```diff
@@ -496,14 +496,12 @@ makeColMap (Teuchos::RCP<const Tpetra::Map<LO, GO, NT> >& colMap,
   return errCode;
 }
 
-template<typename GO, typename device_t>
+template<typename GOView, typename bitset_t>
```

Comment:
These changes are fine. 
Were they needed for HIP, or are they just cleaner code?  If needed for HIP, why?

Reply:
Cleaner code. It was easier to just template on the bitset type than reconstruct it with a typedef in the functor.